README.md

```diff
@@ -133,6 +133,7 @@ Method                                          | Description
 `lessThanEq($value, $value2, $message = '')`    | Check that a value is less than or equal to another
 `range($value, $min, $max, $message = '')`      | Check that a value is within a range
 `oneOf($value, array $values, $message = '')`   | Check that a value is one of a list of values
+`inArray($value, array $values, $message = '')` | Check that a value is one of a list of values (alias of `oneOf`)
```

Comment:
Id say that `oneOf` is an allias for `inArray`.

Reply:
True. Done.